exoscale/api/compute.py

```diff
@@ -2420,6 +2548,45 @@ def get_anti_affinity_group(self, name=None, id=None):
 
         return anti_affinity_groups[0]
 
+    ## Deploy Target
+
+    def list_deploy_targets(self, zone):
+        """
+        List Deploy Targets.
+
+        Parameters:
+            zone (Zone): the zone to list in
+
+        Yields:
+            DeployTarget: the next Deploy Target
+        """
+
+        _list = self._v2_request("GET", "/deploy-target", zone.name)
+
+        for i in _list["deploy-targets"]:
+            yield DeployTarget._from_api(i, zone)
+
+    def get_deploy_target(self, zone, name=None, id=None):
+        """
+        Get a Deploy Target.
+
+        Parameters:
+            name (str): a Deploy Target name
+            id (str): a Deploy Target unique identifier
+
+        Returns:
+            DeployTarget: a Deploy Target
+        """
+
+        if id is None and name is None:
+            raise ValueError("either id or name must be specifed")
+
+        for dt in self.list_deploy_targets(zone):
```

Comment:
we should use `get-deploy-target` endpoint https://github.com/exoscale/schemas/blob/master/resources/schemas/public-api.edn#L740

Reply:
Good catch, I fixed it for all V2 API calls (Deploy Targets, Instance Pools & NLB). Thanks!